[Bug]: Wrong import for discord_api_types in "build/*/decorators/classes/DApplicationCommand.d.ts"

### What happened?

After setting up the project using "create-discordx", I added the `@discordx/music` package and I tried to build a docker image via the Dockerfile, and I encountered the following error. Afterwards, I tried to run `npm run build` and I got the same error.

```TYPESCRIPT
> leu-bot@0.0.1-SNAPSHOT build
> tsc

node_modules/discordx/build/cjs/decorators/classes/DApplicationCommand.d.ts:2:104 - error TS2307: Cannot find module 'discord-api-types' or its corresponding type declarations.

2 import type { APIInteractionDataResolvedChannel, APIInteractionDataResolvedGuildMember, APIRole } from "discord-api-types";

node_modules/discordx/build/esm/decorators/classes/DApplicationCommand.d.ts:2:104 - error TS2307: Cannot find module 'discord-api-types' or its corresponding type declarations.

2 import type { APIInteractionDataResolvedChannel, APIInteractionDataResolvedGuildMember, APIRole } from "discord-api-types";
```

I checked the "discord-api-types" repo and it says we can **only** import the module by specifying the API version.

![image](https://user-images.githubusercontent.com/35991042/164780225-21284b36-4fc8-4fbd-a9cc-301901ffb8fa.png)

### Reproduction

### 1 - Install discordx via the [create-discordx](https://github.com/oceanroleplay/discord.ts/tree/main/packages/create-discordx#-introduction) package.
- Choose npm as the package manager
- Pick the starter template

### 2 - CD into the project
### 3 - Run `npm i @discordx/music`
### 4 - Run `docker build -t bug .`

### Package

@discordx/music

### Version

Stable

### Relevant log output

```Shell
> leu-bot@0.0.1-SNAPSHOT build
> tsc

node_modules/discordx/build/cjs/decorators/classes/DApplicationCommand.d.ts:2:104 - error TS2307: Cannot find module 'discord-api-types' or its corresponding type declarations.

2 import type { APIInteractionDataResolvedChannel, APIInteractionDataResolvedGuildMember, APIRole } from "discord-api-types";

node_modules/discordx/build/esm/decorators/classes/DApplicationCommand.d.ts:2:104 - error TS2307: Cannot find module 'discord-api-types' or its corresponding type declarations.

2 import type { APIInteractionDataResolvedChannel, APIInteractionDataResolvedGuildMember, APIRole } from "discord-api-types";
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/packages/discordx/src/decorators/classes/DApplicationCommand.ts
+++ b/packages/discordx/src/decorators/classes/DApplicationCommand.ts
@@ -13,7 +13,7 @@ import type {
   APIInteractionDataResolvedChannel,
   APIInteractionDataResolvedGuildMember,
   APIRole,
-} from "discord-api-types";
+} from "discord-api-types/v9";
 
 import type {
   ApplicationCommandDataX,

